### PR TITLE
Feat: P0 product list

### PR DESCRIPTION
**Pull request by Naveenchandar**
Product List and Filter

Screenshot 2022-02-12 at 6 14 27 PM

### CODE REVIEW

File: product-list.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+@import url(./variable.css);
+
+body {
+    background: var(--bg-color);
+}
+
+.filter_products {
+    flex-basis: 250px;
```
Comment (aniketsingh13) [inline product-list.css]: any reason for using px here just want to know...

**Review by aniketsingh13 — APPROVED** [review]
good to go

File: product-list.css (hunk 1 above)
Comment (Naveenchandar) [reply product-list.css]: > any reason for using px here just want to know...

No, I've changed it to rem. 🤔

**Review by rohanmathur91 — APPROVED** [review]
Looks good to me

**Pull request merged**